setup.py

```diff
@@ -17,11 +17,11 @@ def load_readme():
 
 setup(
     name='perfflowaspect',
-    description='A tool to easily orchestrate general computational workflows '
-    'both locally and on supercomputers.',
+    description='tool to analyze cross-cutting performance concerns of '
+                'composite scientific workflows.',
     version="0.0.1",
     author='Dong H. Ahn, Stephen Herbein, James Corbett',
-    author_email='ahn1@llnl.gov, herbein@llnl.gov, corbett8@llnl.gov',
+    author_email='ahn1@llnl.gov, herbein1@llnl.gov, corbett8@llnl.gov',
```

Comment:
OK. I see this changes -- great. Like I said, please add your info and then squash this intermediate change into the first commit.